lib/Extension/Bundle/PimcoreBundleManager.php

```diff
@@ -500,10 +500,7 @@ protected function resolvePaths(string $type, string $mode = null): array
     /**
      * Emits given path event
      *
-     * @param array  $paths
-     * @param string $eventName
-     *
-     * @return array
+     * @return string[]
      */
     protected function resolveEventPaths(array $paths, string $eventName): array
```

Comment:
```suggestion
    /**
     * @param string[] $paths
     */
    protected function resolveEventPaths(array $paths, string $eventName): array
```

Reply:
There is already a php doc. I will add it there.

Reply:
> There is already a php doc. I will add it there.

Thanks 👍

Reply:
See: https://github.com/pimcore/pimcore/pull/13996/commits/e48c1d3f2a9170909dd083428e96b03269ccba2b